doc/api/cli.md

```diff
@@ -1272,6 +1272,12 @@ added: v19.6.0
 The destination for the corresponding test reporter. See the documentation on
 [test reporters][] for more details.
 
+### `--test-child-process`
```

Comment:
I am -1 on this implementation, 
CLI flags which are a public API should not be added to solve an internal node core issue
I would prefer to add an environment variable instead.
in addition, @cjihrig  has expressed in the past a concern regarding introducing big differences between `node test.js` and `node --test test.js` - which I agree with, but this case might justify it - it makes more sense to me than parsing CLI flags out of `NODE_OPTIONS`

Reply:
You're right, an environment variable makes more sense, since this flag would only be used for the specific case of a parent test process spawning a child, there's no reason to add extra options to the CLI. I'll do a quick update.